src/Storage.gs

```diff
@@ -0,0 +1,115 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Contains classes used to persist data and access it.
+ */
+
+/**
+ * Creates a new storage instance.
+ * @param {string} prefix The prefix to use for keys in the properties and cache.
+ * @param {PropertiesService.Properties} properties The properties instance to use.
+ * @param {CacheService.Cache} opt_cache The optional cache instance to use.
```

Comment:
Please use [brackets](http://usejsdoc.org/tags-param.html#optional-parameters-and-default-values) for optional params. `@param {CacheService.Cache} [opt_cache]`

Also consider using `cache` rather than `opt_cache` since many param can be falsy and it's not common to see the prefix `opt_`.

Reply:
Done